src/CentreonRemote/Domain/Service/TaskService.php

```diff
@@ -148,7 +148,8 @@ public function getRemoteStatusByParent(int $parentId, string $serverIp, string
             $result = unserialize($remoteDataResult[0]->params);
 
             $httpMethod = $result['params']['http_method'];
-            $httpPort = $result['params']['http_port'];;
+            $httpPort = $result['params']['http_port'];
+            ;
```

Comment:
is this line needed ?

Reply:
I used php cs fixer, I guess it moved it down instead of delete the extra semicolon :)

Reply:
removed